workers/data_refinery_workers/downloaders/management/commands/queue_test_task.py

```diff
@@ -0,0 +1,47 @@
+from django.core.management.base import BaseCommand
+from data_refinery_models.models import (
+    SurveyJob,
+    Batch,
+    BatchStatuses,
+    DownloaderJob
+)
+from data_refinery_workers.downloaders.array_express \
+    import download_array_express
+
+
+# Import and set logger
+import logging
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+# Just a temporary way to queue a celery task
+# without running the surveyor.
```

Comment:
This is one small nitpicky thing that I went back on forth on for a little bit - I think in general we want comments that describe "the operation of ... [a] function or class" in the form of a docstring (as these docs state: http://python-guide-pt-br.readthedocs.io/en/latest/writing/documentation/#docstrings-versus-block-comments), and reserve block comments for things that add extra information for a programmer reading the code (if I am understanding this correctly). However, since this is just a temporary way to do this, I do not feel strongly about it having to be in docstring versus block comments, so it is up to you if you would like to change it or not.